CHANGELOG.md

```diff
@@ -6,9 +6,11 @@
 - [#335] The session expiry for the OIDC provider is now taken from the Token Response (expires_in) rather than from the id_token (exp)
 
 ## Breaking Changes
+- [#440](https://github.com/pusher/oauth2_proxy/pull/440) Switch Azure AD Graph API to Microsoft Graph API (@johejo)
```

Comment:
Would be good to expand on this and add a comment as to why this is a breaking change, similar to as we have done on previous releases